### DISCUSSION TITLE
Mangler det en Title størrelse?

### DISCUSSION THREAD

**Discussion by kennidenni** [fremtind/jokul#2994] [Annet]
Vi har sett litt på font i vår nye kjøpsløsning og har sett at Title er litt stor og Heading-1 er litt liten. 
![image](https://user-images.githubusercontent.com/31621649/182550581-86daa942-959d-48f6-83b7-26429b7c0294.png)

Vi hadde tidligere to Title versjoner, men det ble fjernet ved forrige Jøkul-uke. 

Det ble besluttet å ta opp diskusjonen når noen som jobbet med dette Jøkul-uke er tilbake til jobb.

**Comment by shvoldum** [comment]
En vri på spørsmålet; Er det for stort hopp mellom Title og H1? Burde H1 vært sizet mer som en bro mellom Title og H2 enn den er i dag?

Skudd fra hofta-forslag til endring:
<img width="1190" alt="Skjermbilde 2022-08-02 kl  14 46 22" src="https://user-images.githubusercontent.com/17295531/184880895-3cbfb99d-c2e3-411c-a0dd-eb4ff5da37bd.png">

**Comment by Stakset** [comment]
Vi så på dette i VD-forum og kom fram til dette forslaget. Kom gjerne med innspill 

![Dekstop skala](https://user-images.githubusercontent.com/72978199/187191462-0e08267e-5969-41c0-a0ac-55bfac85a5b1.png)

![Dekstop skala-1](https://user-images.githubusercontent.com/72978199/187191459-65ca21e6-076c-4869-a377-b2107da898d4.png)

![Mobil skala](https://user-images.githubusercontent.com/72978199/187191468-d784cac7-164d-4418-aa3a-a354aea88263.png)

![Mobil skala eksempel](https://user-images.githubusercontent.com/72978199/187191463-35a57198-6d99-4cb2-ab73-1cb56bad97b5.png)

**Reply by wkillerud** [reply]
Publisert i dag i Figma som [Components/Title/Small](https://www.figma.com/file/4LxyVaUrdIQEgDlYxsrqPt/fix%3A-title%2Ftitle-small?node-id=594%3A4252), og i `@fremtind/jkl-core@10.1.0` som `@include jkl.text-style("title-small")` (via #3045)

Cc @kennidenni @shvoldum @Stakset
